scripts/LVGLImage.py

```diff
@@ -308,12 +308,11 @@ def write_c_array_file(
         stride: int,
         cf: ColorFormat,
         filename: str,
+        oname: str,
```

Comment:
Please define full parameter name to avoid "mind-mapping" of names.
It could be: `outputname`